homeassistant/components/overkiz/sensor.py

```diff
@@ -126,7 +126,6 @@ class OverkizSensorDescription(SensorEntityDescription):
     OverkizSensorDescription(
         key=OverkizState.CORE_FOSSIL_ENERGY_CONSUMPTION,
         name="Fossil Energy Consumption",
-        device_class=SensorDeviceClass.ENERGY,
```

Comment:
Why remove the device class? The unit of measurement was missing.

Reply:
Do we know the unit of measurement for sure? I removed the unit of measurement for the once I don't know, since it is not documented by Overkiz :(.